shared/teams/team/settings/index.js

```diff
@@ -0,0 +1,271 @@
+// @flow
```

Comment:
This `render` stayed the same, all the state is stored in this component rather than in recompose hocs